cloud/aws/eks.md

```diff
@@ -1,11 +1,24 @@
 ---
 layout: page
-title: "Amazon Elastic Container Service for Kubernetes (Amazon EKS)"
+title: "Dynamic Provisioning on Amazon Elastic Container Service for Kubernetes (Amazon EKS)"
 keywords: Amazon, portworx, container, Kubernetes, storage, Docker, k8s, pv, persistent disk, eks
 sidebar: home_sidebar
 ---
 
+![k8s porx Logo](/images/k8s-porx.png){:height="188px" width="188px"}
+
 * TOC
 {:toc}
 
-## Coming soon
+
+## Create a EKS cluster
+
+Creating EKS clusters can be found [here](https://aws.amazon.com/eks/).
+
+The steps below will help you enable dynamic provisioning of Portworx volumes in your Amazon Elastic Container Service for Kubernetes (Amazon EKS) cluster.
```

Comment:
Don't repeat Amazon. So just have EKS in the brackets.

Reply:
Updating it but that's how Amazon itself posting in their site. https://aws.amazon.com/eks/

Reply:
Sorry, in that case, let's maintain their branding. You can keep what you had before.